src/README.md

```diff
@@ -126,6 +126,14 @@ implemented the app instance is discarded and a new instance is created.
 *Optional*. Print a report of the current app status.
 
 
+— Method **myapp:start**
+
+*Optional*. Starts the app.
+
+At this point links and app name are set. The app has a chance to do
+additional initialization if needed.
```

Comment:
Part of the design or spec should be what additional things you can do in start() compared to the constructor.  In your code, the links aren't set up, so we can't do linky things; dunno.

To expand on @dpino's use case -- we need the app name and we think waiting for the first push() is kinda hacky :)